### ISSUE TITLE
It's probably time to revisit Robo-stats. Or RUDRA. I honestly don't know which one.

### ISSUE THREAD

**Issue by SkankinGarbage** [psywombats/mgne#745]
I started my Robo-runthrough tonight, and to my complete shock, beating RUDRA was nearly impossible.

I want to show you some things that I tried:

![Screenshot 2022-07-24 22 41 37](https://user-images.githubusercontent.com/60168946/180714217-dbfcfe79-31ab-48c5-a107-3e94e5ae723d.png)

This is how I first arrived at the boss. Besides the SMG being glitched at the time of this playthrough (does not grant any STR), this obviously failed because you just can't budget for STR and AGI with a robot. No big deal - I tried something weird and it didn't work out. So, just replace those with some STR items and take another shot, right?

![Screenshot 2022-07-24 23 10 52](https://user-images.githubusercontent.com/60168946/180719453-4f6fb6ef-9296-4bb6-a537-b208081e4dfe.png)

...Except, I had to get THIS much STR just to hang. I tried doing this with LONG swords, and it failed - the accuracy is just too low. Even this variation barely even worked, as you can see - this was at least my second try (if you miss the TRISHULAs too many times, you have no chance). I still missed a ton, but if you're not in the range of ~20 STR, you just can't hope to be accurate enough to win before you die, and this is just an absurd amount of money.

(NOTE: I know that you are not balancing for low-grind same-race runs; however, I want to note that this had never been a problem through the entire development cycle of the game. I don't know what's changed.)

Here are some ways I attempted to cheese the boss:

![Screenshot 2022-07-24 23 46 42](https://user-images.githubusercontent.com/60168946/180715132-e4ba9e00-ab97-4103-a75a-ca0360b531b9.png)

This is, surprisingly, the most AGI you can get on a Robot at the end of World 1 while still having them equipped with a RAPIER. ...They do like 18-20 damage, it's terrible and a LONG bow is more effective. So, Ninjabot is out.

After that, I had the idea of, "If Ninjabot doesn't work, what about Berserkerbot?"

![Screenshot 2022-07-24 23 36 40](https://user-images.githubusercontent.com/60168946/180715335-d35406d3-4285-4b77-9be8-4ae2c06981dc.png)

...I mean. I won. But, I can't really call this a victory. Especially when this leaves the party very unprepared to move forward in the game.

TL;DR: There doesn't seem to be any way you can actually gear a 4-Robot party to make RUDRA a winnable fight with anything but amazing luck. There could be a few reasons for this:

1. Robo-stats are just too low. This has been something I've worried about for a very long time, but I've also successfully finished Robo-runthroughs, so while I favor revisiting the current state of their stat gains, it makes me wonder if something has changed about the RUDRA fight in particular.

2. RUDRA's and the TRISHULAs' Defense is too high. I don't know if this is really the way to deal with this - Espers don't have any issue getting their stats high enough to fight RUDRA, and Monsters can cheese the fight with Poison and STENCH/ELECTRO (I can never remember which one is available at this point in the game, but that's how I do this fight with Monsters). Plus, I use a very similar strategy with Humans and it works (NOTE: I will do a Human run first thing tomorrow to make sure this is still accurate - stand by while I verify this statement).

3. RUDRA and the TRISHULAs' Agility is too high. The frequency at which I am missing 2-3 attacks per turn is bewildering. This seems like one way to deal with the problem. Mind you, Robots can't get any higher than ~20 STR or AGI by the end of this world - they can't even overtake optimal Human stats for this world with cheesy builds, so thie boss's AGI doesn't need to be all that high.

4. RUDRA and the TRISHULAs' HP is too high. I'm gonna revisit this fight with various different groups, but I wonder if this fight is just always a slog? I can't remember. Will check.

**Comment by psywombats** [comment]
Yeah this group probably doesn't need the AGI they have right now (everyone has 15). 

I'll try a mono-robot playthrough sometime soon but I'd probably approach this fight like I did with mono-humans: stack defense and have a shield available for 1st member, go with musket/strength/ninja from back row, have 1-2 X-POTIONS equipped somewhere, and (crucially for parties with 0 mana) ignore at least 1 trishula. Equipping potions to robots feels bleh, but an x-potion is +13 HP at least, and dropping a +150 heal onto the frontline tank can double the time you're standing in this fight.

**Comment by psywombats** [comment]
also no idea why SMG was missing the +4 str it should've been granting

**Comment by psywombats** [comment]
Okay so O-BLIND and O-CONFUSE have forever had a +8 defense from somewhere that were making this fight stupidly hard (essentially +8 more defense than was intended for trishulas and rudra). This was reducing low(er) damage melee weapons to ineffectiveness, especially noticeable with stuff like longbows that were doing 1-3 damage when they should've been doing 13-15 (def is applied at ^1.15 exponent). I think that's most of what was causing things to drag here. My test party below got through things without needing to heal.
![image](https://user-images.githubusercontent.com/1907581/180942123-2537b844-3120-402e-b080-e977041a05cf.png)

Leaving this open as I think nerfing AGI a bit here (and buffing def a bit now maybe) might be the play. Missing kind of feels shitty with all the ESP in this fight and the inaccurate musket but not sure what to do there for now.

**Comment by SkankinGarbage** [comment]
I went through again and beat it.

![Screenshot 2022-07-27 23 22 35](https://user-images.githubusercontent.com/60168946/181435059-3ed14a6a-0b76-4d5c-9c3c-2ff7f7341673.png)

What's frustrating to me is that I did a slightly different version of this that had no X-Potions...

![Screenshot 2022-07-27 23 23 55](https://user-images.githubusercontent.com/60168946/181435271-9ee330a1-1271-4956-b17e-09f83fd46621.png)

...And this version couldn't win, either by killing a TRISHULA and staying in phase 1, OR bursting them both down and going to phase 2 (which btw, I think is still easier for Robot parties just because the outgoing damage is so low and COUNTER doesn't trigger on ranged attacks). It makes me feel like the fight is less about "How did you prepare your Robots?" and more of a "Did you equip some X-Potions?" 

The biggest problem is that the MUSKET and FREEZE miss too much, and they're the hardest-hitting weapons you can get for Robots besides BATTLE axes (which would require budgeting for AGI - a valid strategy, but not for low grind runs). Missing 2-3 attacks per turn just feels awful. 

Lowering the defense helped marginally - the MUSKET and SMG deal about 10-12 more damage per hit. But, since my strategy revolves around using FREEZE guns with at least two of the party members, it didn't have a huge impact overall. SMG was an all-star in this fight, thanks to the high damage and accuracy; but, there's no way I can realistically budget for more than one.

This can be fixed by lowering AGI on the boss, but I wonder if the right play isn't just to increase the accuracy of these guns? The fight is much harder for low-level Robot/Human parties than it is for low-level Esper/Monster parties, and I think their reliance on guns to deal damage is a big reason why. You could also add more treasure chests with weapons - this is a big part of what makes World 3 possible, for example, and there are a few places where there's little to no treasure that some could be added (Hot Springs only has a shard, Mirror Shrine has a MUSKET at the beginning but all the other dead-end paths don't lead to any treasure).

**Comment by psywombats** [comment]
Making the following balance changes:
 - MUSKET accuracy buffed to 75 (from 65) -- for reference, COLT is 80
 - FIRE gun acc 75 -> 85
 - FREEZE gun acc 65 -> 80
 - VULCAN gun acc 60 -> 75
 - MISSILE cannon acc 50 -> 75
 - BAZOOKA cannon acc 60 -> 75
